src/edit_distance.jl

```diff
@@ -0,0 +1,116 @@
+"""
+Computes the edit distance between graphs G₁ and G₂.
+
+Returns the minimum edit cost and edit path to transform graph
+G₁ into graph G₂. An edit path consists of a sequence of pairs
+of vertices (u,v) ∈ [0,|G₁|] × [0,|G₂|] representing vertex
+operations:
+
+- (0,v): insertion of vertex v ∈ G₂
+- (u,0): deletion of vertex u ∈ G₁
+- (u>0,v>0): substitution of vertex u ∈ G₁ by vertex v ∈ G₂
+
```

Comment:
what about edge's insertions/deletions/substitutions and their associated cost? https://en.wikipedia.org/wiki/Graph_edit_distance